scanpy/external/exporting.py

```diff
@@ -450,7 +450,7 @@ def cellbrowser(
         Coordinates missing from `adata` are skipped.
         By default (or when specifying `'all'` or `None`), these keys are tried:
         [`"tsne"`, `"umap"`, `"pagaFa"`, `"pagaFr"`, `"pagaUmap"`, `"phate"`,
-        `"fa"`, `"fr"`, `"kk"`, `"drl"`, `"rt"`].
+        `"fa"`, `"fr"`, `"kk"`, `"drl"`, `"rt"`, `"trimap"`].
```

Comment:
Nice catch. @all: we should create this string using code, not by requiring people to find and update it.